components/admin/AdminNavBar.tsx

```diff
@@ -67,6 +67,7 @@ const AdminNavBar: React.FC = () => {
                   {/* Replace the label and route with your menu items */}
                   <Tab label="Dashboard" onClick={() => navigationToScreens("./dashboard")}  />
```

Comment:
You have to update the same in drawer component as well.

Reply:
Content tab added to drawer components.